src/store/amplify-config.ts

```diff
@@ -0,0 +1,63 @@
+import type { OAuthConfig } from "@aws-amplify/core";
+import type { ResourcesConfig } from "aws-amplify";
+import { IS_TEST } from "constants/env";
+import { appRoutes } from "constants/routes";
+
+const common = IS_TEST
```

Comment:
this is only temporary as current `prod` doesn't have any auth in it. We will eventually switch back to auto-generated `aws-exports` file where `master` would have it's own config from `amplify pull prod` and `staging` from `amplify pull dev`